packages/block-library/src/columns/column.js

```diff
@@ -1,12 +1,66 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { Path, SVG } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { InnerBlocks } from '@wordpress/block-editor';
+import { InnerBlocks, BlockControls, BlockVerticalAlignmentToolbar } from '@wordpress/block-editor';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
 
 export const name = 'core/column';
 
+const ColumnEdit = ( { attributes, updateAlignment } ) => {
+	const { verticalAlignment } = attributes;
+
+	const classes = classnames( 'block-core-columns', {
```

Comment:
Was this intentionally named as the plural "columns" ? This is for the single "column" block, a separate block from the plural "columns" block. Should it not then be named `'block-core-column'` instead?